src/utils/formatter.js

```diff
@@ -147,6 +147,13 @@ export const formatExpiry = prevExpiry => {
       return `${expiry[0]} / `;
     }
   }
-  return expiry.join(' / ');
+
+  expiry = expiry.join(' / ');
```

Comment:
Let's yank out the spaces here. It'll give us some room with the sizing issues we've been having.